string shipment_type
string agv_id
string station_id
nist_gear/Product part
nist_gear/Model part_in_world
int32 total_parts
int32 priority
